Week3/homework/homework.sql

```diff
@@ -0,0 +1,160 @@
+  `Emp_Phone` varchar(50) NOT NULL,
+  `Emp_Email` varchar(50) NOT NULL,
+  `Emp_Addres` varchar(50) NOT NULL,
+  KEY `Emp_Number` (`Emp_Number`),
+  CONSTRAINT `Emp_Info_ibfk_1` FOREIGN KEY (`Emp_Number`) REFERENCES `Employee` (`Emp_Number`)
+) ENGINE=InnoDB DEFAULT CHARSET=latin1;
+/*!40101 SET character_set_client = @saved_cs_client */;
+
+--
+-- Dumping data for table `Emp_Info`
+--
+
+LOCK TABLES `Emp_Info` WRITE;
+/*!40000 ALTER TABLE `Emp_Info` DISABLE KEYS */;
+/*!40000 ALTER TABLE `Emp_Info` ENABLE KEYS */;
+UNLOCK TABLES;
+
+--
+-- Table structure for table `Employee`
+--
+
+DROP TABLE IF EXISTS `Employee`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!40101 SET character_set_client = utf8 */;
+CREATE TABLE `Employee` (
+  `Emp_Number` int(11) NOT NULL AUTO_INCREMENT,
+  `Emp_Salary` double NOT NULL,
+  `Emp_HiringDate` date NOT NULL,
+  `Emp_Birth` date NOT NULL,
+  `Dept_Number` int(11) NOT NULL,
+  PRIMARY KEY (`Emp_Number`),
+  KEY `Dept_Number` (`Dept_Number`),
+  CONSTRAINT `Employee_ibfk_1` FOREIGN KEY (`Dept_Number`) REFERENCES `Departement` (`Dept_Number`)
+) ENGINE=InnoDB DEFAULT CHARSET=latin1;
+/*!40101 SET character_set_client = @saved_cs_client */;
+
+--
+-- Dumping data for table `Employee`
+--
+
+LOCK TABLES `Employee` WRITE;
+/*!40000 ALTER TABLE `Employee` DISABLE KEYS */;
+/*!40000 ALTER TABLE `Employee` ENABLE KEYS */;
+UNLOCK TABLES;
+
+--
+-- Table structure for table `Employee_Project`
+--
+
+DROP TABLE IF EXISTS `Employee_Project`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!40101 SET character_set_client = utf8 */;
+CREATE TABLE `Employee_Project` (
+  `Emp_Number` int(11) NOT NULL,
+  `Project_Number` int(11) NOT NULL,
+  PRIMARY KEY (`Emp_Number`,`Project_Number`),
+  KEY `Project_Number` (`Project_Number`),
+  CONSTRAINT `Employee_Project_ibfk_1` FOREIGN KEY (`Emp_Number`) REFERENCES `Employee` (`Emp_Number`),
+  CONSTRAINT `Employee_Project_ibfk_2` FOREIGN KEY (`Project_Number`) REFERENCES `Project` (`Project_Number`)
+) ENGINE=InnoDB DEFAULT CHARSET=latin1;
+/*!40101 SET character_set_client = @saved_cs_client */;
+
+--
+-- Dumping data for table `Employee_Project`
+--
+
+LOCK TABLES `Employee_Project` WRITE;
+/*!40000 ALTER TABLE `Employee_Project` DISABLE KEYS */;
+/*!40000 ALTER TABLE `Employee_Project` ENABLE KEYS */;
+UNLOCK TABLES;
+
+--
+-- Table structure for table `Project`
+--
+
+DROP TABLE IF EXISTS `Project`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!40101 SET character_set_client = utf8 */;
+CREATE TABLE `Project` (
+  `Project_Number` int(11) NOT NULL AUTO_INCREMENT,
+  `Project_Name` varchar(50) NOT NULL,
+  `Project_Location` varchar(50) NOT NULL,
+  `Dept_Number` int(11) NOT NULL,
+  PRIMARY KEY (`Project_Number`),
+  UNIQUE KEY `Project_Name` (`Project_Name`),
+  KEY `Dept_Number` (`Dept_Number`),
+  CONSTRAINT `Project_ibfk_1` FOREIGN KEY (`Dept_Number`) REFERENCES `Departement` (`Dept_Number`)
```

Comment:
Is it necessary to associate a project with a department, when projects are associated with employees who are also associated with departments?